### DISCUSSION TITLE
Editing Taterzens with commandbloxks?

### DISCUSSION THREAD

**Discussion by LostLea** [samolego/Taterzens#135] [General]
I just wanna ask, can i edit a Taterzen with a commandblock and if how?
Im not sure since u cant pick the npc you want to edit.
So if i want the Npc to go to a place what do i put into the Commandblock?
pls help i love your mod <33

**Comment by LostLea** [comment]
I want my npc to go to a place at a specific time of day. Thats why i need the commandblock to tell the npc to go there at a specific time of day if u know what i mean

**Reply by samolego** [reply]
You can /execute as  <uuid> run npc ...

**Reply by LostLea** [reply]
I know but i need to select a specific npc dont i?

**Reply by LostLea** [reply]
im typing "execute as Fiona run npc action goto 224 77 238 but she wont go there what am i doing wrong???

**Reply by samolego** [reply]
Use uuid instead of name

**Comment by LostLea** [comment]
Thx i love ur service :}
